### ISSUE TITLE
Reloading properties files

### ISSUE THREAD

**Issue by callain** [spring-projects/spring-loaded#128]
Hello,

Can you support the reload of properties files like i18n etc ?
It seems to not working when i add a message and use it directly in my JSP.

**Comment by aclement** [comment]
Yes, it does only watch class files right now (and most recently jars of classes too) - but it can theory watch anything/everything for changes.

**Comment by callain** [comment]
It would be nice, it is especially a pain since spring loaded doubles the start time of my webapp

**Comment by jama707** [comment]
yeah, it would be great if properties file could be reloaded!!
